mrp_bom_produce_delay/model/mrp_bom.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (C) 2024 - Today: GRAP (http://www.grap.coop)
+# @author: Quentin DUPONT (quentin.dupont@grap.coop)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class MrpBom(models.Model):
+    _inherit = "mrp.bom"
+
+    produce_delay = fields.Float(
```

Comment:
```suggestion
    lead_time = fields.Float(
```
delay is "retard" !?

Reply:
produce_delay is the field in MRP in Product model !
I also don't understand it's technical english name (delay is "retard" in FR) but the help message is more clear
That why i take the logic of the core field  : https://github.com/odoo/odoo/blob/16.0/addons/mrp/models/product.py#L33

![image](https://github.com/user-attachments/assets/b672cd85-537d-4534-9220-282f1e5bfe26)

Reply:
Yes in product form you need string 'Manuf' because there are many purposes but in bom view manufacture vocabulary is implicit, don't repeat yourself, KISS.

Reply:
ok for technical name